### PR TITLE
Solve some security vulnerabilities and bump version to 6.1.2

### PR DESCRIPTION
**Pull request by Jazper**
Before:
`166 vulnerabilities (1 low, 66 moderate, 67 high, 32 critical)`

After:
`10 vulnerabilities (7 moderate, 3 high)`

Runtime - `0 vulnerabilities`
All tests are green.

### CODE REVIEW

**Comment by th0r** [comment]
> since there are tests for older versions of webpack

Let's remove them then if we don't use those versions already

**Comment by Jazper** [comment]
Lowered issues to -
`19 vulnerabilities (14 moderate, 5 high)`

All `high` issues are in the latest versions of the packages. I think we can leave it like that for now.

File: examples/browser-sprite-extract-mode/webpack.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = merge(baseConfig, {
         test: /\.svg$/,
```
Comment (th0r) [inline examples/browser-sprite-extract-mode/webpack.config.js]: Let's just remove this examples folder - it doesn't make sense to us. Will it reduce number of dev deps?

**Review by th0r — CHANGES_REQUESTED** [review]

File: examples/browser-sprite-extract-mode/webpack.config.js (hunk 1 above)
Comment (Jazper) [reply examples/browser-sprite-extract-mode/webpack.config.js]: Removed examples folder. This reduced the number of problems -
`14 vulnerabilities (11 moderate, 3 high)`

Comment (th0r) [reply examples/browser-sprite-extract-mode/webpack.config.js]: Why do we need `react` dependency now, after removing an `examples` folder? Is it used anywhere else?
That's why I asked "Will it reduce number of dev deps".

Could you carefully go through all the dependencies and remove the unused ones please.

Comment (Jazper) [reply examples/browser-sprite-extract-mode/webpack.config.js]: I hope I removed all unused code and packages.

**Review by th0r — APPROVED** [review]

File: examples/browser-sprite-extract-mode/webpack.config.js (hunk 1 above)
Comment (th0r) [reply examples/browser-sprite-extract-mode/webpack.config.js]: Great, thanks 👍

**Comment by th0r** [comment]
Published `@workato/svg-sprite-loader@6.1.2`

**Pull request merged**